### PR TITLE
Run two tabs at a time with threading

### PR DESCRIPTION
**Pull request by es50678**
I'm newish to python. I want to basically make this so you can pass in args for how many tabs or bowsers you want running at a time.

IE:
```
python req.py --parrallel 5
```
would run 5 tabs or browsers at a time

### CODE REVIEW

File: req.py [hunk 1]
```diff
@@ -1,334 +1,351 @@
...
+            driver.close()
+            continue
+
+        driver.close()
+        time.sleep(5)
+
+
+if __name__ == '__main__':
+    t1 = threading.Thread(target=main)
+    t2 = threading.Thread(target=main)
```
Comment (pws1453) [inline req.py:340]: Great implementation. One thing to note: python's threading module will not allow you to run both of threads simulataneously. Sadly, only multiprocessing and futures support this. You can read about this here. https://docs.python.org/3/library/threading.html

**Review by pws1453 — COMMENTED** [review]
Functionality provided by threading is better implemented in https://github.com/SeanDaBlack/KelloggBot/pull/21. This should not be pulled. However, this is functional code and a great way to learn the basics of threading.

**Comment by BigweldIndustries** [comment]
#21 Is now closed due to being incredibly behind on pulls. I will create a new pull requests implementing this some time later today

**Comment by BigweldIndustries** [comment]
This approach is deprecated and further discussion should be brought to #39

**Comment by es50678** [comment]
@BigweldIndustries  @pws1453 thanks for the feedback! Checking out #39 for learning and inspiration. Appreciate being a part of this <3

**Pull request closed without merging**